Modules/IO/BMP/src/itkBMPImageIO.cxx

```diff
@@ -70,7 +70,6 @@ BMPImageIO::CanReadFile(const char * filename)
   if (!extensionFound)
   {
     itkDebugMacro(<< "The filename extension is not recognized");
-    return false;
```

Comment:
The same behavior is found for  BMPImageIO, JPEGImageIO, JPEG2000ImageIO, MINCImageIO, MRCImageIO, NrrdImageIO, StimulateImageIO, but not for other IO, like TiffImageIo...